experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/PopulationSizing/index.tsx

```diff
@@ -0,0 +1,286 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { SizingByUserType } from "@mozilla/nimbus-schemas";
+import React from "react";
+import { Form } from "react-bootstrap";
+import { Code } from "src/components/Code";
+import LinkExternal from "src/components/LinkExternal";
+import TooltipWithMarkdown from "src/components/PageResults/TooltipWithMarkdown";
+import { ReactComponent as Info } from "src/images/info.svg";
+
+type PopulationSizingProps = {
+  sizingData: SizingByUserType;
+  totalNewClients: number;
+  totalExistingClients: number;
+};
+
+const popSizingHelpMarkdown =
+  "Pre-computed sizing is computed with the [auto-sizing](https://github.com/mozilla/auto-sizing) tool. This assumes a power requirement of 0.8 and is based on Application, Channel, Locale (or Language), and Country (and is only available for some common combinations of these).";
+
+const PopulationSizing = ({
+  sizingData,
+  totalNewClients,
+  totalExistingClients,
+}: PopulationSizingProps) => {
+  const newUserSizing = sizingData.new;
+  const existingUserSizing = sizingData.existing;
+  const targetingParameters = JSON.stringify(
+    newUserSizing.target_recipe,
+    (k, v) => (k === "new_or_existing" ? undefined : v),
+  );
+  return (
+    <>
+      <Form.Label
+        className="d-flex align-items-center"
+        data-testid="population-sizing-precomputed-values"
+      >
+        Pre-computed population sizing data
+        <Info
+          data-tip
+          data-for="auto-sizing-help"
+          width="20"
+          height="20"
+          className="ml-1"
+        />
+        <TooltipWithMarkdown
+          tooltipId="auto-sizing-help"
+          markdown={popSizingHelpMarkdown}
+        />
+      </Form.Label>
+      <div>
+        <p className="text-secondary">
+          This is meant as a guide for certain targeting parameters. As always,
+          contact Data Science in{" "}
+          <LinkExternal href="https://mozilla.slack.com/archives/CF94YGE03">
+            #ask-experimenter
+          </LinkExternal>{" "}
+          for assistance.
+        </p>
+      </div>
+      <Form.Group>
+        <Form.Label className="d-flex align-items-center">
+          Targeting Parameters
+        </Form.Label>
+        <Code codeString={targetingParameters} />
+      </Form.Group>
+      <Form.Label
+        className="d-flex align-items-center"
+        data-testid="new-total-clients-label"
+      >
+        {totalNewClients} total{" "}
+        <span
+          style={{ fontWeight: 700, marginLeft: "0.3em", marginRight: "0.3em" }}
```

Comment:
Oh try to use the [bootstrap layout classes](https://getbootstrap.com/docs/3.3/css/#grid) if you can 🙏